<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>maps api</string>
  </property>
  <widget class="QLabel" name="map_label">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>75</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>map</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="theme_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>75</y>
     <width>20</width>
     <height>455</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>73</y>
     <width>20</width>
     <height>458</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>66</y>
     <width>453</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>521</y>
     <width>456</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Приближение - PageUp/PageDown, Управление - стрелки</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>type object's name</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>13</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>task8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск - Enter, Выделить поле - Tab/Esc</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="addressEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
